src/registrar/tests/data/test_domain_contacts.txt

```diff
@@ -0,0 +1,8 @@
+Anomaly.gov|ANOMALY|tech
```

Comment:
**suggestion (blocking)**: Same issue with actual usernames occurring here.

Reply:
This is totally fake data.  Is there another consideration I am missing?

Reply:
(We spoke about this on slack.  Resolving in next push)